### PR TITLE
protect from subprocess failure

### PR DESCRIPTION
**Pull request by falseperfection**
Tests were failing because of an unhandled exception.

See: https://jira.mesosphere.com/browse/INFINITY-2231

### CODE REVIEW

File: conftest.py [hunk 1]
```diff
@@ -44,7 +38,8 @@ def get_task_ids(user: str=None):
 def get_task_logs_for_id(task_id: str, lines: int=1000000):
     try:
         task_logs = subprocess.check_output([
-            'dcos', 'task', 'log', task_id, '--lines', str(lines)]).decode()
+            'dcos', 'task', 'log', task_id, '--lines', str(lines), 'stderr'
```
Comment (benclarkwood) [inline conftest.py]: We should capture both

cc @falseperfection

Comment (NimaVaziri) [inline conftest.py]: FWIW we don't currently use stdout for anything. Only somewhat useful info is confirming all expected artifacts are fetched properly.

File: conftest.py [hunk 2]
```diff
@@ -42,8 +36,13 @@ def get_task_ids(user: str=None):
...
 def get_task_logs_for_id(task_id: str, lines: int=1000000):
-    return subprocess.check_output([
-        'dcos', 'task', 'log', task_id, '--lines', str(lines)]).decode()
+    try:
+        task_logs = subprocess.check_output([
+            'dcos', 'task', 'log', task_id, '--lines', str(lines), 'stderr'
+        ]).decode()
+        return task_logs
+    except subprocess.CalledProcessError:
+        return None
```
Comment (elezar) [inline conftest.py]: Note that we now fail silently. Would some logging output be useful to see what caused the failure?

File: conftest.py (hunk 1 above)
Comment (benclarkwood) [inline conftest.py]: The scheduler nearly exclusively logs to stdout.

Comment (falseperfection) [inline conftest.py]: No worries: I'll get them both.

File: conftest.py (hunk 2 above)
Comment (falseperfection) [inline conftest.py]: 👍

File: conftest.py (hunk 1 above)
Comment (NimaVaziri) [inline conftest.py]: Right, I was thinking non-scheduler tasks for some reason...

**Pull request merged**